src/test/java/fi/metatavu/kuntaapi/test/server/integration/ptv/TestPtvConsts.java

```diff
@@ -2,76 +2,351 @@
+      "7cf9a776-808e-4e58-a904-863f48887ea2",
+      "610325e0-f485-4caa-bd3b-5bf74d5480c5",
+      "47e7c82b-9d67-4a7e-93f7-c6ff87adf2fa",
+      "a683d5eb-ee1a-4869-98ba-a4a00ed880da",
+      "9e63c517-812a-42a0-9e5b-3c1a0f0ff630",
+      "d2ea56fe-1bca-4d26-8579-875a4c6c9638",
+      "b7fb6633-d754-45fb-8241-35f22b75c2dd"
+    }
+  };
+
+  public final static String[][] SERVICE_ELECTRONIC_CHANNEL_CHANNELS = {
+    { "3e0951ab-890f-441b-bcf5-5c27b56b1fde" },
+    { "a165743c-4121-4ab5-b1d4-e04ff36ee430" },
+    { "80f6bcb5-e224-4041-97da-95b6936878fe" },
+    { "80f6bcb5-e224-4041-97da-95b6936878fe" },
+    { "7374dd89-8e26-4402-90fe-c32942525476" },
+    { },
+    { "3e0951ab-890f-441b-bcf5-5c27b56b1fde" },
+    { },
+    { "80f6bcb5-e224-4041-97da-95b6936878fe" },
+    { "b8931c4b-1119-4c69-9bc4-a41fb00b92fe" },
+    { "b8931c4b-1119-4c69-9bc4-a41fb00b92fe" },
+    { "6631f997-eb58-401e-b0a9-274f7d51cb0a" },
+    { "3e0951ab-890f-441b-bcf5-5c27b56b1fde" },
+    { },
+    { "daee70a8-ec2d-413d-ab79-2ebf79fe04a3" },
+    { "2f648147-a615-4d31-b251-f4fbe3a26ee4", "3e0951ab-890f-441b-bcf5-5c27b56b1fde" },
+    { "1d01e828-362b-4cd2-b302-c3a4b8b1941c" },
+    { "b8931c4b-1119-4c69-9bc4-a41fb00b92fe" },
+    { "3e0951ab-890f-441b-bcf5-5c27b56b1fde" },
+    { "3e0951ab-890f-441b-bcf5-5c27b56b1fde" },
+    { "3e0951ab-890f-441b-bcf5-5c27b56b1fde" },
+    { },
+    { "3e0951ab-890f-441b-bcf5-5c27b56b1fde" },
+    { "3e0951ab-890f-441b-bcf5-5c27b56b1fde" },
+    { },
+    { },
+    { "b8931c4b-1119-4c69-9bc4-a41fb00b92fe" },
+    { "3e0951ab-890f-441b-bcf5-5c27b56b1fde" },
+    { },
+    { "80f6bcb5-e224-4041-97da-95b6936878fe" }
+  };
+
+  public final static String[][] SERVICE_SERVICE_LOCATION_CHANNELS = {
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { "1bd39cda-3006-482e-acff-93ab8ddc9f1b", "41ebe689-dbed-4bb2-a6f2-184e29d625ed" },
+    { },
+    { },
+    { "abac25e1-bd15-4c1a-b22e-9d9a705d9220" },
+    { },
+    { },
+    { },
+    { },
+    { "13992280-9c95-4ede-af13-33ad2acde746", "5d04a7cd-d372-409e-b3d6-211ae5325bd2", "ecdf1bc2-ad32-46cc-9421-50ce61bc710f" },
+    { },
+    { "2481937a-5d4d-4643-a9b9-9877be5d4d3e", "3cfd7097-b73e-4f2a-a482-099b00f12b02", "69c2f81a-bc28-4992-9491-b82dacc0868c" },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { "32833c27-7d41-4267-800a-a9a888b513b7" },
+    { },
+    { },
+    { }
+  };
+
+  public final static String[][] SERVICE_PRINTABLE_FORM_CHANNELS = {
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { "7de53057-287b-4899-b4b3-86291bf47b45", "c38feebb-9dfa-4efc-b2f7-02b2fd282841" },
+    { },
+    { },
+    { },
+    { },
+    { },
+    { "14a9874a-8b9b-488a-aec5-5321dd36678f" },
+    { }
+  };
+
+  public final static String[][] SERVICE_PHONE_CHANNELS = {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this member "protected". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2386)